backend/src/org/akvo/lumen/lib/dashboard.clj

```diff
@@ -0,0 +1,29 @@
+(ns org.akvo.lumen.lib.dashboard
+  (:require [org.akvo.lumen.lib.dashboard-impl :as impl]))
+
+
+(defn all
+  "Returns all dashboards."
+  [tenant-conn]
+  (impl/all-dashboards tenant-conn))
+
+(defn create
+  "Creates a new dashboard. Body should include keys and spec conform
```

Comment:
Agree, I'll remove that section. We should add the information once we have something.